packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/Convert.java

```diff
@@ -862,6 +795,15 @@ static List<LatLng> toPoints(Object o) {
     return points;
   }
 
+  static List<LatLng> toPoints(List<Messages.PlatformLatLng> data) {
```

Comment:
Please call this `pointsFromPigeon`, matching the style of some of the existing methods, so that we don't have two things with the same name, one of which takes `Object`, because it makes it hard to clearly reason about which one is being called since `List<Messages.PlatformLatLng>` is also an `Object`. Having clearly separable codepaths makes the `Object` versions less of a foot-gun for as long as we have to have them.